src/libproc_macro/lib.rs

```diff
@@ -80,7 +524,11 @@ pub struct LexError {
 /// all of the contents.
 #[unstable(feature = "proc_macro_internals", issue = "27812")]
 #[doc(hidden)]
+#[path = ""]
```

Comment:
`mod quote;` in `__internal`; this allows the file to be `libproc_macro/quote.rs`. Thinking about this some more, I'll move `mod quote` into the crate root and remove the `#[path = ""]`.